mern-ecommerce/controllers/auth.js

```diff
@@ -73,10 +73,12 @@ exports.isAuth = (req, res, next) => {
 };
 
 exports.isAdmin = (req, res, next) => {
-  if (req.profile.role === 0) {
-    return res.status(403).json({
-      error: 'Admin resource! Access denied',
+  if (req.profile.role == 0) { 
+    res.status(403); 
```

Comment:
`res.status(403);` should be accompanied by a response message to clearly communicate the reason for the access denial. It's recommended to include a response message along with setting the status code. For example, `res.status(403).json({ error: 'Access denied' });`.